### ISSUE TITLE
M2.1 "Invalid Form Key" when saving configurable product with multiple configurations

### ISSUE THREAD

**Issue by 5zymon** [magento/magento2#5405]
When I try to create or edit configurable product that has multiple simple products in configurations, Magento returns "Invalid Form Key. Please refresh the page." error. Of course refreshing the page, clearing cookies, cache, nothing works. When I limit number of configurations to a lot less (for example 4) same form comes trough just fine. This issue was not present on 2.0.7 I was able to save configurable products with multiple configurations. 
![1_2_ct _t w _diamond_ladies_engagement_ring_14k_white_gold_products_inventory_products_magento_admin_-_2016-07-01_14 00 11](https://cloud.githubusercontent.com/assets/3308234/16521101/d6b4061c-3f94-11e6-8aa9-d222582bc077.png)
![1_2_ct _t w _diamond_ladies_engagement_ring_14k_white_gold_products_inventory_products_magento_admin_-_2016-07-01_13 59 34](https://cloud.githubusercontent.com/assets/3308234/16521102/d6d7063a-3f94-11e6-98fa-2713436df900.png)

**Comment by 5zymon** [comment]
The solution is to increase max_input_vars in php.ini file. not a Magento 2 issue.

**Comment by SewHappy58** [comment]
I tried increasing PHP input to 7500 and it still doesn't work in Magento 2.1

**Comment by bpgkt** [comment]
use php_value[max_input_vars] = 10000
instead of max_input_vars

**Comment by jpcsguru** [comment]
This does not work

**Comment by ricardowong** [comment]
max_input_vars on php.ini worked for me on Magento 2.1.6

**Comment by miteshdhaduk** [comment]
It does not work in Magento2.2.2

**Comment by neilgreene** [comment]
Magento 2.2.3 and I have tried increasing max_input_vars in php.ini, I have cleared var/cache, browser cache, Magento cache, verified that for my magento.conf and magento-ssl.conf that both are using the same domain, removed domain cookies.   Still, get the error when trying to edit an existing product.  But, I can create new products without any issue.  

Will to share hitting my head on the wall if anyone is interested.
